cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/FormInputSections/ClusterNameInput.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ChangeEvent } from 'react';
+import { CdsControlMessage, CdsFormGroup } from '@cds/react/forms';
+import { CdsInput } from '@cds/react/input';
+
+export function ClusterNameSection(
+    field: string,
```

Comment:
Suggestion: - For Args instead of using the params if we use object params. It will be compile error if we misplace the arguments.  
Ex: Here we have errors and register as type any.  we could easily pass register , errors in args and compiler wont show error.  Instead we could have function ClusterNameSection({field, errors, register, onEnterClusterName})

Reply:
Also a good point. I think I should either do the object approach you use as an example, or correctly type the `errors` and `register` params